core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -214,7 +214,8 @@
       mode.
     */
     html, body {
-      background: url('/images/general/background.jpg') no-repeat center center fixed;
+      /*TODO(gvishal): modify image url to use cache slugs*/
```

Comment:
Should be ok. Maybe add a TODO somewhere, too, and file an issue; we should find a way to sort this out properly.